numpy/testing/_private/utils.py

```diff
@@ -876,6 +880,10 @@ def assert_array_equal(x, y, err_msg='', verbose=True):
         The error message to be printed in case of failure.
     verbose : bool, optional
         If True, the conflicting values are appended to the error message.
+    strict : bool, optional
+        If True, raise an assertion when either the shape or the data
```

Comment:
Either "raise an exception" or "raise an AssertionError" (not "raise an assertion").